### PR TITLE
L2 perf

### PR DESCRIPTION
**Pull request by jtaleric**

### CODE REVIEW

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.18-nightly-x86-udn-density-l2-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.18-nightly-x86-udn-density-l2-24nodes

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.18-nightly-x86-udn-density-l2-24nodes

File: ci-operator/step-registry/openshift-qe/udn-density-l2-pods/openshift-qe-udn-density-l2-pods-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    documentation: |-
+      Set to true if the job is for baremetal
+  - name: ITERATION_MULTIPLIER_ENV
+    default: "9"
+    documentation: |-
+      The number of iterations per worker nodes to create
+  - name: CHURN
+    default: "false"
+    documentation: |-
+      Default is false for udn-density-l3-pods workload
```
Comment (krishvoor) [inline ci-operator/step-registry/openshift-qe/udn-density-l2-pods/openshift-qe-udn-density-l2-pods-ref.yaml:28]: Please update references of l3 to l2

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 2]
```diff
@@ -4082,6 +4082,82 @@ periodics:
...
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: aws-4.18-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.18"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.18-nightly-x86-udn-density-l2-24nodes
+  spec:
```
Comment (krishvoor) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:4104]: Please add the slack reporter_config:

```
  reporter_config:
    slack:
      channel: '#ocp-qe-scale-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :white_check_mark: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :white_check_mark:
        {{else}} :warning:  Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> :warning: {{end}}'
```

**Review by krishvoor — COMMENTED** [review]
Minor nits

**Review by rsevilla87 — APPROVED** [review]
/lgtm

**Pull request merged**